src/utils/MSFraggerAdapter.cpp

```diff
@@ -189,7 +191,20 @@ class TOPPMSFraggerAdapter final :
 
 
   TOPPMSFraggerAdapter() :
-    TOPPBase("MSFraggerAdapter", "Peptide Identification with MSFragger", false,
+    TOPPBase("MSFraggerAdapter",  "Peptide Identification with MSFragger.\n"
+                                  "Important note:\n"
+                                  "The Regents of the University of Michigan (“Michigan”) grants us permission to redistribute the\n"
+                                  "MS Fragger application developed by Michigan within the OpenMS Pipeline and make available\n"
+                                  "for use on related service offerings supported by the University of Tubingen and the Center for\n"
+                                  "Integrative Bioinformatics.\n"
+                                  "Per the license agreement the use of the pipeline and associated materials is for academic\n"
+                                  "research, non-commercial or educational purposes. Any commercial use inquiries\n"
+                                  "must be directed to the University of Michigan Technology Transfer Office at\n"
+                                  "techtransfer@umich.edu. All right title and interest in MS Fragger shall remain with the\n"
+                                  "University of Michigan.\n"
+                                  "\n"
+                                  "Please see the supplied license file License.txt for details.\n"    
```

Comment:
How are users gonna find this txt in the different distributions?

Reply:
I think we need to do two things:
- We need to distribute the EULA as txt file
- Make sure that the user agreed during installation or execution

I could, of course, also include the EULA text into the wrapper, but I am not sure if this is
legally sufficient.

Reply:
Can you create an issue for that for the release?

Reply:
let's do that when we include MSFragger in the thirdparty